verification/curator-service/api/openapi/openapi.yaml

```diff
@@ -500,11 +503,154 @@ components:
       type: object
           # TODO: Define NewSource object.
     Case:
-      description: A complete case; a superset of "#/components/schemas/NewCase" with server-added fields
-      allOf:
-        - $ref: "#/components/schemas/NewCase"
-        - type: object
-          # TODO: Define Case object.
+      description: A single line-list case.
+      properties:
+        caseReference:
+          type: object
+          properties:
+            sourceId:
+              type: string
+              pattern: '^[a-f\d]{24}$'
+            sourceUrl:
+              type: string
+            additionalSources:
+              type: array
+              items:
+                type: object
+                properties:
+                  sourceUrl:
+                    type: string
+        demographics:
+          type: object
+          properties:
+            gender:
+              type: string
+            ageRange:
+              type: object
+              properties:
+                start:
+                  type: number
+                end:
+                  type: number
+            ethnicity:
+              type: string
+            nationalities:
+              type: array
+              items:
+                type: string
+            occupation:
+              type: string
+        location:
+          $ref: "#/components/schemas/Location"
+        events:
+          description: > 
+            An event with name "confirmed" must be specified. Other event names
+            include onsetSymptoms, firstClinicalConsultation, selfIsolation,
+            hospitalAdmission, icuAdmission, and outcome.
```

Comment:
Not a change to make now, but if we're dictating the title of events, I wonder if the `name` should be an enum. Or for lack of that, if this is the best place to document the "standard" value set.

Reply:
yeah enum probably makes sense -- add a todo